.ci/ppc64le/kubernetes/init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 ARM Limited
```

Comment:
2021?

Reply:
I copied the file from the aarch64 directory, so I left the copyright notice unchanged